content/tracing/advanced_usage/_index.md

```diff
@@ -1410,24 +1410,32 @@ The purpose of this section is to explain how the correlation between traces and
 
 Leverage the MDC Frameworks ([Map Diagnostic Context][1]) to automatically correlate trace and span IDs into your logs. Datadog MDC keys may be injected automatically with a tracer integration, or manually injected with the Datadog API.
 
-**Enable Log Injection for JSON Formatted Logs**
+There are 3 different implementations possible depending on the logging configuration:
```

Comment:
```suggestion
There are three possible implementations depending on the logging configuration:
```

The rules around rendering numbers in English are complex and, frankly, inconsistent across style guides. A simple rule that everyone can agree one is this: _small_ numbers should be written out. 😄